### DISCUSSION TITLE
Multiplayer maps

### DISCUSSION THREAD

**Discussion by mefistotelis** [swfans/syndwarsfx#124] [General]
While SW Port does not support multiplayer, it is possible to start the MP maps alone.

Select IPX - that protocol has Ethernet card communication disabled, so it will allow you to start the game alone.

Since unused teams are removed from MP maps, there isn't much to do in such games - but just wanted to mention that it is working.

**Comment by Moburma** [comment]
Yeah, works pretty good. I had it crash once when I exited a game (for those not in the know you have to press Shift+escape to exit MP games), but generally ok. It only lets you play one game and then stops working, but I expect you know that. 
One thing I did was extend the .ini file to include the missing maps from all098.miss. Some of these conflict, so you only get three more maps in the end, but I think worth having. The only problem is they are mostly.... weird. Bahrain doesn't work great as the game puts loads of weapons in the ocean. That said The Moon (click on columbo orbit station) looks hugely entertaining, each player starts off next to one of those claw mechs.

I stuck it here for now:

https://pastebin.com/jepB9tCT

Oh, and for those not in the know, you need to set the tech and money levels before it unlocks the map screen where you can choose which map you want, otherwise I think it just sticks you on a random one.
